api/v1beta1/controlplane_types.go

```diff
@@ -48,6 +54,8 @@ type ControlPlaneSpec struct {
 	Glance GlanceSpec `json:"glance,omitempty"`
 	// Placement API settings
 	Placement PlacementSpec `json:"placement,omitempty"`
+	// AMQ Interconnect settings
+	AMQInterconnect AMQInterconnectSpec `json:"placement,omitempty"`
```

Comment:
c/p missing replace -> json:"placement

Reply:
Is there a concern about using the `AMQInterconnect` name which is downstream only? Upstream it's just called `qdr-operator` with an `Interconnect` object type.  https://github.com/interconnectedcloud/qdr-operator

Reply:
@stuggi ack on the json:placement correction. Fix incoming...

@liefmadsen I'm actually pretty happy to continue using AMQInterconnect for now. Our immediate target is "make this easy to install on OpenShift". As the "AMQ Interconnect" operator is in Operator Hub for OpenShift proper (downstream) its just plain easier to install. Correct me if I'm wrong but if we switch to QDR I'd need to add in at least an extra catalogsource for this to work right? Eventually we may switch to the QDR route but I'm not sure we care where it comes from right now and as downstream is easier for our use case its probably better.

Reply:
Sorry, I didn't mean to suggest you stop using AMQ Interconnect itself, as I agree, it's great, and the upstream isn't easy to consume. I have the same problem. 100% just use the downstream Operator.

However in the naming conventions, I would avoid using `AMQInterconnect` and rather just use the object name that AMQ Interconnect / QDR Operator satisfies, which is `Interconnect`.

That way you're not mixing naming conventions and embedding downstream product names into upstream projects.

Reply:
I won't call out every use of `amq-interconnect`, `AMQInterconnect` etc, but I would suggest renaming them all to just `interconnect`.

Reply:
@leifmadsen The latest patch renames everything to Interconnect. I did a squash and push force so we are in sync. Thanks for the suggestions.